main.js

```diff
@@ -4,6 +4,10 @@ var board = function () {
   var theBoard = [[5,4,3,2,1],[],[]];
   var numberOfMoves = 0;
 
```

Comment:
I'd suggest next time not scoping all of your functionality to one function. The module pattern is appropriate but you can divide your logic into sub-modules. i.e. a module for housing your board and it's state, and then one for manipulating the board, etc. This pattern would be more re-usable and allows your code to be more readable.